### DISCUSSION TITLE
마이크 켜고 끄는 단축키가 있으면 좋겠습니다.

### DISCUSSION THREAD

**Discussion by jihwanKo** [metamong-friends/metamong-friends.github.io#20] [제안하기]
환경에 따라 하울링은 완전히 피할 수 없는 경우도 있을거라, 
마이크를 끄고 켜기 쉽도록 단축키가 있으면 좋을거 같습니다.

**Comment by ACT-TJ** [comment]
현재 이모지만 단축키를 설정해 두었는데 사용성을 위해 단축키 추가는 필요할 것 같습니다.
다만 키가 많아지면 유저 인지시키는 고민을 하고 있고,
키 안내 페이지도 같이 릴리즈 할 수 있도록 voc 로 관리하도록 하겠습니다.
의견 너무 감사합니다.
